extensions/powershell/README.md

```diff
@@ -18,28 +18,31 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 - Please don't edit this section unless you're re-configuring how the powershell extension plugs in to AutoRest
 AutoRest needs the below config to pick this up as a plug-in - see https://github.com/Azure/autorest/blob/master/docs/developer/architecture/AutoRest-extension.md
 
-
 #### PowerShell
-``` yaml
-
-enable-multi-api: true
-load-priority: 1001
 
+> Requires remodeler and csharpv2
+``` yaml 
 use-extension:
   "@microsoft.azure/autorest.remodeler": "beta"
   "@microsoft.azure/autorest.csharp-v2": "beta"
 ```
 
-
-``` yaml $(powershell)
+> forces multi-api mode 
+``` yaml
 enable-multi-api: true
+load-priority: 1001
+```
+
+# Pipeline Configuration
```

Comment:
I put these values here, since we *really* don't want them in autorest-core (that'd be hard to fix later) and if they are here, they get set before the c# generator runs, and it has to pick up a couple of these in order to work right.

Reply:
The reason to not have these values here is because you have duplicated defaults defined. I can understand putting api-folder and runtime-folder here, but the others are defaulted in the code for the PowerShell generator. We can discuss.